gtsfm/evaluation/metrics_report.py

```diff
@@ -134,13 +169,67 @@ def get_figures_for_metrics(metrics_group: GtsfmMetricsGroup) -> Tuple[str, str]
                 raise ValueError(f"Metric {metric_name} does not contain a summary.")
             # Add a scalar metric for mean of 1D distributions.
             scalar_metrics["mean_" + metric_name] = value[metrics.SUMMARY_KEY]["mean"]
+            scalar_metrics["median_" + metric_name] = value[metrics.SUMMARY_KEY]["median"]
```

Comment:
This might be too verbose, I suggest we use either median or mean.

Reply:
Thanks, it is now just median.